crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -418,7 +426,11 @@ impl<M: NetworkMsg, K: SignatureKey + 'static> Libp2pNetwork<M, K> {
     }
 
     /// Initiates connection to the outside world
-    fn spawn_connect(&mut self, id: usize) {
+    fn spawn_connect<const MAJOR: u16, const MINOR: u16>(
+        &mut self,
+        id: usize,
+        bind_version: StaticVersion<MAJOR, MINOR>,
```

Comment:
question to check my understanding: we're just using `bind_version` just to propagate the `MAJOR` and `MINOR` types down the chain right?